docs/runbooks/VirtControllerRESTErrorsBurst.md

```diff
@@ -1,20 +1,57 @@
+<!-- Edited by Jiří Herrmann, 7 Nov 2022 -->
+
 # VirtControllerRESTErrorsBurst
 
 ## Meaning
 
-More than 80% of the rest calls failed in virt-controller for the last 5 minutes
+More than 80% of REST calls in `virt-controller` pods failed in the last 5 minutes.
+
+The `virt-controller` has likely fully lost the connection to the API server.
+
+This error is frequently caused by one of the following problems:
+
+- The API server is overloaded, which causes timeouts. To verify if this is the case, check the metrics of the API server, and view its response times and overall calls.
+
+- The `virt-controller` pod cannot reach the API server. This is commonly caused by DNS issues on the node and networking connectivity issues.
 
 ## Impact
 
-virt-controller has potentially fully lost connectivity to the apiserver. Running workloads will not be impacted but status updates will not be propagated and actions like migrations can not take place.
+Status updates are not propagated and actions like migrations cannot take place. However, running workloads are not impacted. 
 
 ## Diagnosis
 
-There exist two common types of errors:
-- The apiserver is overloaded and we run into timeouts.
-  Issues like this can be identified by checking the apiserver metrics and looking at its response times, overall calls, …
-- The virt-controller pod can’t reach the apiserver. Common issues are: dns issues on the node, networking connectivity issues
+Check whether `virt-controller` can connect to the API server.
+
+1. Set the `NAMESPACE` environment variable::
```

Comment:
Typo - `:` x 2
Don't forget to unindent the code blocks.
Remove any blank lines between lead-in and code block.